CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.1.4] - 2020-10-23
```

Comment:
You don't need to do it mannualy, just remove this line. Our bot will do it by itself